google-http-client/src/test/java/com/google/api/client/util/OpenCensusUtilsTest.java

```diff
@@ -36,9 +40,11 @@
  */
 public class OpenCensusUtilsTest extends TestCase {
 
-  @Rule public ExpectedException thrown = ExpectedException.none();
   TextFormat mockTextFormat;
   TextFormat.Setter mockTextFormatSetter;
+  TextFormat originTextFormat;
+  TextFormat.Setter originTextFormatSetter;
+  Span mockSpan;
```

Comment:
Why do you not use real mock objects? Like use mockito?

Reply:
It seems that most of the test mocks are written manually so I just followed that convention, Besides all the tests are written in JUnit3 style.

But yeah I think that is a good idea. I'll rewrite that.